drivers/ipm/ipm_mcux.c

```diff
@@ -20,8 +20,14 @@
 #define MCUX_IPM_DATA_REGS 1
 #define MCUX_IPM_MAX_ID_VAL 0
 
-#if (defined(LPC55S69_cm33_core0_SERIES) || defined(LPC55S69_cm33_core1_SERIES))
-#ifdef LPC55S69_cm33_core0_SERIES
+#if (defined(LPC55S69_cm33_core0_SERIES) || defined(LPC55S69_cm33_core1_SERIES) || \
+defined(MCXN947_cm33_core0_SERIES) || defined(MCXN947_cm33_core1_SERIES) || \
+defined(MCXN946_cm33_core0_SERIES) || defined(MCXN946_cm33_core1_SERIES) || \
+defined(MCXN547_cm33_core0_SERIES) || defined(MCXN547_cm33_core1_SERIES) || \
+defined(MCXN546_cm33_core0_SERIES) || defined(MCXN546_cm33_core1_SERIES))
```

Comment:
Can we use `CONFIG_SOC_SERIES_MCXN` instead of the above list?

Reply:
Yes we can, Thanks for the tip.